### PR TITLE
Add - ticpos function

### PR DESCRIPTION
**Pull request by TestSubjector**
* TODO.md: Remove 'ticpos' from list.
* docs/src/ref.md: Add "ticpos" entry to the manual.
* src/utils.jl: Add ticpos entry to "utils".
* src/ticpos.jl: Contains ticpos function.
* test/util-tests.jl: Include tests for ticpos and minor stylistic fix.

### CODE REVIEW

File: src/ticpos.jl [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _ticpos{T<:AbstractFloat}(deglen::T, pixlen::T, ticsize::T)
+    minpix = deglen*60/pixlen
+    incr = minpix*ticsize
+
+    if incr < 0
```
Comment (giordano) [inline src/ticpos.jl]: You can replace the whole `if` with
```julia
sgn = copysign(1, incr)
```

File: src/ticpos.jl [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    if incr >= 30
+        units = "Degrees"
+    elseif incr > 0.5
+        units = "Arc Minutes"
+    else
+        units = "Arc Seconds"
+    end
+
+    if incr >= 120
+        incr = 4
```
Comment (giordano) [inline src/ticpos.jl]: Mind type-stability!  ;-)  Above, `incr` is an `AbstractFloat`, here you assign it an `Integer` value, causing instability.  You can either introduce a new variable (`incr1` or whatever) to which assign this integers, or keep `incr` but cast the integers to `T` (e.g., with `T(4)` in this line).

It really takes some exercise to do this right, I often find type-instabilities in my old code (last time happened to me just today)

File: src/ticpos.jl [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        incr = 1
+    end
+
+    if units == "Degrees"
+        minpix = minpix/60
+    elseif units == "Arc Seconds"
+        minpix = minpix*60
+    end
+
+    ticsize = incr/abs(minpix)
```
Comment (giordano) [inline src/ticpos.jl]: I'm not sure the absolute value is necessary here (not your fault, it's there in original code): `minpix` is the ratio between two lengths, that should be positive-definite.

Comment (TestSubjector) [inline src/ticpos.jl]: Yeah, that portion did seem odd.
However if we are assuming that minpix should positive, then we don't need to use the sgn-if block (for which you recommended)  or even `incr = abs(incr)` (as ticsize should be positive too). So remove those portions too?

Comment (giordano) [inline src/ticpos.jl]: Right.  Indeed all argument look as  if they should be positive.  Maybe specify this in the docstring that the argument are assumed to be positive.

**Pull request merged**